apps/meteor/client/lib/chats/uploads.ts

```diff
@@ -55,14 +56,27 @@ const send = async (
 	]);
 
 	try {
+		// Assuming that only description is available in case of attachments
+		const message = description
+			? await onClientBeforeSendMessage({
+					_id: id,
+					msg: description,
```

Comment:
I don't think we should do this, it would be clearer if we add the support for encrypting file descriptions in the functions instead of using the `msg` field instead.